lib/diagnostic.js

```diff
@@ -46,7 +46,7 @@ let person2 = {
 };
 
 // Question 3:  Invoke the `fullName()` method so it returns `Santa Clause`?
-let answer3 = null; // replace null with your response
+let answer3 = fullName.call(person2); // replace null with your response
```

Comment:
`fullName` is a method on `person1`, not a function in the global scope, so this will not work.  You need to call `person1`'s `fullName()` method but use `Function`'s `call()` method to call it in the context of `person2`.

```javascript
person1.fullName.call(person2);
```